### ISSUE TITLE
French accessibility

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/report-a-cybercrime#1896]
## Summary

1. The html lang attribute is only set to "en". French Voiceover cannot function properly for francophone users without the lang attribute set to "fr"
2. The page title is permanently set to "Report a scam"

## Steps to reproduce

1. Using voiceover in french, in the french version of the app
2. Navigate through the app. Some text will be read with the french voice, interactables will be read in english
3. The page title remains untranslated.

## Unresolved questions

Route paths also remain untranslated. Is there a way to create translation for those?

**Comment by ngosset** [comment]
This issue may not be true anymore, but if it is it should be addressed.

**Comment by justinr86** [comment]
> This issue may not be true anymore, but if it is it should be addressed.

This is still an issue. Screen reader is reading French text in English.
